python/docs/source/reference/pyspark.errors.rst

```diff
@@ -20,10 +20,34 @@
 Errors
 ======
 
+Classes
+-------
+
+.. currentmodule:: pyspark.errors
+
+.. autosummary::
+    :toctree: api/
+
+    PySparkException
+    CapturedException
```

Comment:
I think we don't need to expose CapturedExceptiom anymore?

Reply:
Yeah, just removed `CapturedExceptiom` from API list